vpr/src/route/rr_graph_storage.h

```diff
@@ -585,7 +623,53 @@ class t_rr_graph_storage {
                             "Attempted to access RR node 'side' for non-IPIN/OPIN type '%s'",
                             rr_node_typename[node_data.type_]);
         }
-        return node_storage[id].dir_side_.side;
+        std::bitset<NUM_SIDES> side_tt = node_storage[id].dir_side_.sides;
```

Comment:
I think you need to detect if more than one side is present, and generate an error if this method is used in that case.

Reply:
No problem. I will throw a fatal error in this case